README.md

```diff
@@ -1,2 +1,126 @@
-# sample_alert_info
-Release notes for sample Rubin Observatory alert packets
+# Rubin Observatory Sample Alerts
+
+The Vera C. Rubin Observatory is releasing sample alert packets produced by its pipelines run on precursor survey data.
+These alerts are intended to provide candidate community alert brokers as well as science users the chance to preview the alert format and contents and to provide feedback to the project.
+
+## Obtaining the Data
+
+Sample alert data may be obtained at [this link](https://lsst.ncsa.illinois.edu/~ebellm/sample_precursor_alerts/).
+We anticipate providing reprocessed and updated alert samples as pipeline development progresses, so the data are arranged in a directory structure:
+
+```
+    sample_precursor_alerts/
+	YYYY-MM-DD/
+	    dataset/
+		all_visits_{dataset}_{date}.avro
+		association_{dataset}_{date}.db
+		single_ccd_sample_{dataset}_{date}.avro
+		single_visit_sample_{dataset}_{date}.avro
+```
+
+### Alert packets
+
+Alerts are provided as Snappy-compressed [Avro](https://avro.apache.org/) files.
+The schema used to store the alerts is included in the file.
+
+As the complete sample is about 1 TB, We include two smaller subsets--alerts from a single CCD-visit, and all of the alerts from all CCDs in a single visit.
+
+### Database
+In addition to accessing alert packets via community brokers, during the LSST survey science users with data rights will be able to query the relational Prompt Products Database (PPDB).  
+The contents of the alerts are taken directly from the PPDB `DIASource`, `DIAForcedSource`, `DIAObject`, and `SSObject` records.
+For some use cases direct queries to the PPDB may be more efficient than processing alert packets.
+
+To allow users to explore these different modes of data access, we are also providing the PPDB (`association_{dataset}_{date}.db`) from which the sample alerts were generated as an [SQLite](https://www.sqlite.org) database.
+The SQLite database can be queried using SQL through a variety of interfaces.
+
+Note that in operations, users will query the PPDB using [ADQL](http://www.ivoa.net/documents/ADQL/) via a [TAP service](http://www.ivoa.net/documents/TAP/).
+
+### Software
+Users should install [lsst.alert.packet](https://github.com/lsst/alert_packet) to parse alerts and optionally [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) and [alert-stream](https://github.com/lsst-dm/alert_stream) if they wish to simulate a Kafka alert stream.
```

Comment:
I think we need a bit more explanation here. In particular:

- https://github.com/lsst-dm/alert-stream-simulator talks about running `make` to download datasets from `~swnelson`. Is that in addition to the datasets described in this file? How do they fit together?
- Both lsst-dm/alert_stream and lsst-dm/alert-stream-simulator talk about using `docker-compose`, etc, to get up and running. Are users supposed to follow both of those sets of instructions? Does one take precedence?

To be fair, I'm not sure exactly what other documentation you plan to provide to end users. If this file is all that's coming, then I think it needs to address questions like the above. If there's a separate “software reference” that explains how these packages fit together, then the above points are moot.

Reply:
> https://github.com/lsst-dm/alert-stream-simulator talks about running make to download datasets from ~swnelson. Is that in addition to the datasets described in this file? How do they fit together?

I vote that we should change `alert-stream-simulator`'s Makefile to get the datasets from Eric's directory.

> Both lsst-dm/alert_stream and lsst-dm/alert-stream-simulator talk about using docker-compose, etc, to get up and running. Are users supposed to follow both of those sets of instructions? Does one take precedence?

Almost everything in `lsst-dm/alert_stream`'s README is obsolete and subsumed by `alert-stream-simulator`, I think.

Reply:
I see this README as focused on the sample alerts themselves; usage instructions for `alert-stream-simulator` etc. should live with those packages.

My suggestion would be:

- I put a symlink at the root of the data directory (`https://lsst.ncsa.illinois.edu/~ebellm/sample_precursor_alerts/`) for the latest single CCD and single visit datasets (and full dataset, I guess), and `alert-stream-simulator` can pull those via the Makefile
- We update the  `alert-stream-simulator` README to describe how to bring in larger alert samples (download them into `data/`, which is simple enough, but explicit instructions are good)
- We wipe out everything in the `alert_stream` README that we're not planning to use or publicly support (at least right now).